utils.py

```diff
@@ -0,0 +1,147 @@
+from typing import Any
+
+import logging
+import requests
+import time
+
+import pandas
+
+
+class EkuboLiquidity:
+    URL = "http://178.32.172.153/orderbook/"
+    DEX = "Ekubo"
+    LOWER_BOUND_VALUE = 0.95
+    UPPER_BOUND_VALUE = 1.05
+
+    def __init__(
+        self,
+        data: pandas.DataFrame,
+        collateral_token: str,
+        debt_token: str,
+    ) -> None:
+        self.data = data
+        self.collateral_token = collateral_token
+        self.debt_token = debt_token
+
+        cleaned_collateral_token = self._remove_leading_zeros(collateral_token)
+        cleaned_debt_token = self._remove_leading_zeros(debt_token)
+
+        self.params_for_bids = {
+            "base_token": cleaned_collateral_token,
+            "quote_token": cleaned_debt_token,
+            "dex": self.DEX,
+        }
+        self.params_for_asks = {
+            "base_token": cleaned_debt_token,
+            "quote_token": cleaned_collateral_token,
+            "dex": self.DEX,
+        }
+
+    def apply_liquidity_to_dataframe(
+        self, bids_or_asks: dict[str, Any]
+    ) -> pandas.DataFrame:
+        """
+        Applying liquidity bids or asks data to dataframe, saving in object and returns data
+        :param bids_or_asks: dict[str, Any]
+        :return: pandas.DataFrame
+        """
+
+        liquidity_dataframe = pandas.DataFrame(
```

Comment:
```suggestion
        liquidity = pandas.DataFrame(
```
.. No need to put the type into the name 😉